docs/building.md

```diff
@@ -134,6 +134,26 @@ export PARTHENON_ROOT=$(pwd)/parthenon
 ```
 We set the latter variable for easier reference in out-of-source builds.
 
+### Default machine configurations
+
+To make the default configuration on widely used systems easier, Parthenon provides machine configuration files that contain default options.
+Defaults options include, but are not limited to setting
+- the compiler (e.g., `nvcc_wrapper` for Cuda builds), or
+- paths to non default package locations (e.g., for a custom HDF5 install), or 
+- custom MPI related commands used in the Parthenon test suite (e.g., the launch command).
+
+The machine configurations shipped with Parthenon are located in [`PARTHENON_ROOT/cmake/machinecfg`](../cmake/machinecfg) and are named by the machine name.
+In order to use them either
+- set the `MACHINE_CFG` environment variable to the appropriate file, or
```

Comment:
If I do `cmake .. -DMACHINE_CFG=path/to/my/CMakeLists.txt -DDISABLE_HDF5=True` or something, will the `-DDISABLE_HDF5` override whatever's in my custom machine file? It should, right?

Reply:
Yes, it should, as long as you're doing `set(DISABLE_HDF5 ON CACHE BOOL "Blah")`. `CACHE` variables do nothing if the variable is already set in cache.